src/desk.sol

```diff
@@ -79,7 +79,7 @@ contract Desk is Switchable {
     }
 
     // --- Desk ---
-    function balance() public auth {
+    function balance() public {
```

Comment:
Removing the `auth` here will cause an unstable version. Which we should be aware of.

Because we currently not support async operations:

If everybody would use the proxies it is still fine.

If a borrower would use the core contracts directly, we have some edge cases
- borrower repays but never picks up his NFT
    - `pile.want()`  => negative and `valve.want()` => 0
    - `balance()` check would always fail

I will open a hackMD for the async feature.

Reply:
@manuelpolzhofer I think that's fine. that would be a feature of the system that we can shield most people from by letting them use the proxies.